docs/FRSKY_R9_TX.md

```diff
@@ -0,0 +1,171 @@
+
+### ELRS Bootloader (Recommended) ###
+
+To flash the ELRS bootloader and mLRS firmware you will need to use a radio with OpenTX &#8805; 2.3.12 or EdgeTX &#8805; 2.4.0.
+
+Notes:
+- If you have never previously flashed the R9M module via ST-Link, you can use the stock Frsky bootloader to flash the ELRS bootloader alongside the stock bootloader. This method will preserve the ability to return to the stock Frsky firmware.
+- Flashing the bootloader only needs to be performed once. 
+- If you experience a "No Sync" error, ensure that you have selected CRSF (with 400k baud) for the External RF module in the radio.
+
+The ExpressLRS documentation provides ELRS specific instructions [here](https://www.expresslrs.org/quick-start/transmitters/frsky-r9modules/#via-stock_bl) which you might want to reference if the instructions here are not clear. Of course, you can ignore the references to the ELRS configurator.
+
+#### Flash the ELRS Bootloader ####
+
+1. Download the [r9m\_elrs\_bl.frk](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/bootloader/r9m_elrs_bl.frk?raw=true) file from the ExpressLRS git repository and copy it to the FIRMWARE folder on your radio's SD card.
+2. Install the R9M module in your radio and power it on.
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Scroll to the FIRMWARE folder and select the r9m\_elrs\_bl.frk file. Choose "Flash external module".
+5. Follow the instructions in the next section to flash the mLRS firmware.
+If you have previously flashed via ST-Link, you can install the ELRS bootloader using ST-Link instead:
+
+1. Download the [r9m\_bootloader.bin](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/bootloader/r9m_bootloader.bin?raw=true) file from the ExpressLRS git repository.
+2. Connect to the R9M and perform a full chip erase.
+3. Flash the bootloader.bin to the beginning of the flash (address 0x8000000).
+4. Follow the instructions in the next section to flash the mLRS firmware.
+
+#### Flash/Update the mLRS Firmware ####
+
+Once the ELRS bootloader is installed, the following precedure can be followed to install/update the mLRS firmware.
+
+1. Download the latest mLRS firmware, or build with MLRS\_FEATURE\_ELRS\_BOOTLOADER defined.
+2. Copy the latest tx-R9M-f103c8-elrs-bl-v*.elrs file to FIRMWARE folder on your radio's SD card. Note the "elrs-bl" in the file name.
+   - Note: On bw radios it will be necessary to rename the firmware file to a shorter file name such as "tx-r9m.elrs"
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Select the FIRMWARE folder and scroll to and select the tx-R9M-f103c8-elrs-bl-v*.elrs file. Choose "Flash external ELRS".
+
+#### Install the mLRS Lua Script ####
```

Comment:
duplicate, is also below
out of sequence too as not on flashing

Reply:
Removed